appinventor/appengine/src/com/google/appinventor/client/DarkModeImages.java

```diff
@@ -0,0 +1,82 @@
+// -*- mode: java; c-basic-offset: 2; -*-
+// Copyright 2009-2011 Google, All Rights reserved
```

Comment:
Since this is a new file we don't need to have the Google copyright here. Also, the MIT copyright would start from 2025.